aria-datepicker/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@astroapps/aria-datepicker",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
This should be 1.2.0, I ran into an dependency issue at one point which was caused by new functionality being included in a fix release so i'm warey of getting this right now